starport/pkg/cosmosutil/gentx.go

```diff
@@ -0,0 +1,95 @@
+package cosmosutil
+
+import (
+	"encoding/json"
+	"errors"
+	"os"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+type (
+	Info struct {
+		DelegatorAddress string
+		PubKey           []byte
+		SelfDelegation   sdk.Coin
+	}
+	StargateGentx struct {
+		Body struct {
+			Messages []struct {
+				DelegatorAddress string `json:"delegator_address"`
+				ValidatorAddress string `json:"validator_address"`
+				PubKey           struct {
+					Key string `json:"key"`
+				} `json:"pubkey"`
+				Value struct {
+					Denom  string `json:"denom"`
+					Amount string `json:"amount"`
+				} `json:"value"`
+			} `json:"messages"`
+		} `json:"body"`
+	}
+	ChainGenesis struct {
+		AppState struct {
+			Auth struct {
+				Accounts []acc `json:"accounts"`
+			} `json:"auth"`
+		} `json:"app_state"`
+	}
+	acc struct {
```

Comment:
Can't see where this is being used?

Reply:
it's used for the test assertion, but I forgot to move to the test package, it's done now